examples/distributed-pytorch/train.py

```diff
@@ -0,0 +1,66 @@
+"""Distributed training example with PyTorch.
```

Comment:
Let's have the python script in a different folder, or have a clear separation from the yaml file. Otherwise, it feels quite confusing: is train.py the actual training script when looking at the folder

Reply:
also, might worth it to include the distributed example in the quickstart tutorial as well, so people can see the benefit of skypilot immediately with multi-node training: https://docs.skypilot.co/en/latest/getting-started/tutorial.html#